src/chainstate/stacks/boot/mod.rs

```diff
@@ -1398,7 +1398,7 @@ pub mod test {
 
                 if cur_reward_cycle >= alice_reward_cycle {
                     // this will grow as more miner rewards are unlocked, so be wary
-                    if tenure_id >= (MINER_REWARD_MATURITY + MINER_REWARD_WINDOW + 1) as usize {
+                    if tenure_id >= (MINER_REWARD_MATURITY + 1) as usize {
```

Comment:
`if tenure_id > MINER_REWARD_MATURITY as usize` would be more readable

Reply:
I'm not to worried about this in tests.  `MINER_REWARD_MATURITY` should be whatever type it gets used with in production code (which is `u64`).